rotkehlchen/api/v1/schemas.py

```diff
@@ -3531,3 +3532,55 @@ class FalsePositveSpamTokenSchema(Schema):
 
 class SpamTokenSchema(Schema):
     token = AssetField(expected_type=EvmToken, required=True)
+
+
+class CalendarCommonEntrySchema(Schema):
+    name = fields.String(required=True)
+    description = fields.String(load_default='')
+    counterparty = fields.String(load_default=None)
+    address = fields.String(load_default=None)
+
+
+class NewCalendarEntrySchema(CalendarCommonEntrySchema):
+    timestamp = TimestampField(required=True)
+
+
+class DeleteCalendarEntrySchema(IntegerIdentifierSchema):
+    ...
+
+
+class UpdateCalendarSchema(NewCalendarEntrySchema, IntegerIdentifierSchema):
+    ...
+
+
+class QueryCalendarSchema(DBPaginationSchema, TimestampRangeSchema, DBOrderBySchema):
+    name = fields.String(load_default=None)
+    description = fields.String(load_default=None)
+    counterparty = fields.String(load_default=None)
+    addresses = fields.List(fields.String, load_default=None)
+    to_timestamp = TimestampField(load_default=None)
```

Comment:
This is already part of `TimestampRangeSchema`

Reply:
Then do not inherit from it and redefine it. 

But this is silly. Just inherit and and not redefine. Keep consistent api defaults across the app. It's up to the frontend what it will call it with.